src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector512.cs

```diff
@@ -1102,7 +1102,7 @@ public static Vector512<ulong> Create(ulong e0, ulong e1, ulong e2, ulong e3, ul
         /// <returns>A new <see cref="Vector512{T}" /> initialized from <paramref name="lower" /> and <paramref name="upper" />.</returns>
         /// <exception cref="NotSupportedException">The type of <paramref name="lower" /> and <paramref name="upper" /> (<typeparamref name="T" />) is not supported.</exception>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static Vector512<T> Create<T>(Vector256<T> lower, Vector256<T> upper)
+        public static Vector512<T> Create<T>(Vector256<T> lower, Vector256<T> upper)
```

Comment:
Why were these all internal before? What changed?

Reply:
They were internal because they weren't "API approved". Making all the types (`Vector64/128/256<T>`, `Vector<T>`, and now `Vector512<T>`) consistent was approved in https://github.com/dotnet/runtime/issues/76593 and so the changes here are just exposing the members that were internal publicly, as well as exposing `get_One`, `scalar divide`, and the `shift operators`.